data/mods/Magiclysm/monsters/demon_spider.json

```
@@ -61,7 +61,7 @@
   {
     "id": "mon_demon_spider",
     "type": "MONSTER",
-    "description": "This spider is the size of a car.  Its huge compound eyes seem to be infinite pools of blackness that seem to suck the light out of the air.  Its maroon carapace is studded with wicked-looking spikes that drip with some kind of viscious black liquid that sizzles when it touches the ground.",
+    "description": "This spider is the size of a car.  Its huge compound eyes look almost like infinite pools of blackness that seem to suck the light out of the air.  Its maroon carapace is studded with wicked-looking spikes that drip with some kind of vicious black liquid that sizzles when it touches the ground.",
```

```suggestion
    "description": "This spider is the size of a car.  Its huge compound eyes seem to be infinite pools of blackness that seem to suck the light out of the air.  Its maroon carapace is studded with wicked-looking spikes that drip with some kind of viscous black liquid that sizzles when it touches the ground.",
```
i don't understand the wording change of `seems to be` because it's correct. also, it's viscous (thick and gloopy) not vicious (mean, angry)

---

"seem" is used again in the same sentence.

---

that's less "fixing a typo" and more "adjusting the prose" but i see your point. `looks to be` is fine? `looks almost like` seems clunky to me

---

```suggestion
    "description": "This spider is the size of a car.  Its huge compound eyes look like infinite pools of blackness that seem to suck the light out of the air.  Its maroon carapace is studded with wicked-looking spikes that drip with some kind of vicious black liquid that sizzles when it touches the ground.",
```
I wasn't calling it a typo, BTW, although I can see the confusion.